app/views/experiment_info.eco

```diff
@@ -10,4 +9,4 @@
     color: #ffffff;
     background-color: rgba(20,11,3,0.50);
     font-size: 13px;
-"><span style="color: #cccccc;"><% if User.current?: %><%= User.current.name %><% else: %>(anonymous)<% end %> (<% if Experiments.currentParticipant? && Experiments.currentParticipant.fallback: %>fallback to control<% else: %><% if Experiments.currentCohort?: %><%= Experiments.currentCohort %><% else: %>no <% end %> cohort<% end %>) likes: </span><span class="value"><% if Experiments.currentParticipant? && Experiments.currentParticipant.most_liked_species?: %><%= Experiments.currentParticipant.most_liked_species.join(', ') %><% else: %>N/A<% end %></span>&nbsp;&nbsp;&gt;&nbsp;&nbsp;<span style="color: #cccccc;">Source: </span><span class="subject-source value"><% if Experiments.sources[@subject.zooniverseId]: %><%= Experiments.sources[@subject.zooniverseId] %><% else: %>N/A<% end %></span></div>
\ No newline at end of file
+"><span style="color: #cccccc;"><% if User.current?: %><%= User.current.name %><% else: %>(anonymous)<% end %> (<% if ExperimentServer.currentParticipant? && ExperimentServer.currentParticipant.fallback: %>fallback to control<% else: %><% if ExperimentServer.currentCohort?: %><%= ExperimentServer.currentCohort %><% else: %>no <% end %> cohort<% end %>) likes: </span><span class="value"><% if ExperimentServer.currentParticipant? && ExperimentServer.currentParticipant.most_liked_species?: %><%= ExperimentServer.currentParticipant.most_liked_species.join(', ') %><% else: %>N/A<% end %></span>&nbsp;&nbsp;&gt;&nbsp;&nbsp;<span style="color: #cccccc;">Source: </span><span class="subject-source value"><% if ExperimentServer.sources[@subject.zooniverseId]: %><%= ExperimentServer.sources[@subject.zooniverseId] %><% else: %>N/A<% end %></span></div>
```

Comment:
Doesn't matter, that file isn't used any more, it's only something I use for testing . will probably delete it in a future commit.